### PR TITLE
Add Meeting class to Model class diagram

### PR DESCRIPTION
**Pull request by Javiery3889**

### CODE REVIEW

File: docs/images/ModelClassDiagram.png
Comment (Dethada) [inline docs/images/ModelClassDiagram.png:1]: The arrow between `AddressBook` and `ModelManager` is somehow misaligned now.

Comment (Dethada) [inline docs/images/ModelClassDiagram.png:1]: the line labels for `Person` is also misaligned now

**Review by Dethada — CHANGES_REQUESTED** [review]

File: docs/images/ModelClassDiagram.png
Comment (Javiery3889) [reply docs/images/ModelClassDiagram.png:1]: @Dethada good catch, fixed in 770803f

**Review by Dethada — APPROVED** [review]

**Review by KohGuanZeh — APPROVED** [review]
LGTM

**Pull request merged**